core/sdk-core/src/main/java/software/amazon/awssdk/core/exception/SdkException.java

```diff
@@ -185,6 +225,25 @@ public String message() {
             return message;
         }
 
+        public Integer getNumAttempts() {
+            return numAttempts;
+        }
+
+        public void setAttemptCount(Integer attemptCount) {
```

Comment:
setNumAttempts?

Reply:
Following the same impl of `writableStackTrace` that includes getters and setters only in this base `SdkException`